src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -366,14 +365,13 @@ def unprotect_subvolume_snapshot(self, **kwargs):
         ret        = 0, "", ""
```

Comment:
nit: Should we also throw deprecated message to console ?
lgtm otherwise

Reply:
Addressed in the latest push